code/edgemedia/src/androidTest/java/com/adobe/marketing/mobile/edgemedia/internal/MediaDBServiceTests.java

```diff
@@ -9,7 +9,7 @@
   governing permissions and limitations under the License.
 */
 
-package com.adobe.marketing.mobile.media.internal;
+package com.adobe.marketing.mobile.edgemedia.internal;
```

Comment:
The package name should be `com.adobe.marketing.mobile.edge.media.internal` to match how we've defined the packages for Edge Identity and Edge Consent.